docs/resources/User.md

```diff
@@ -39,6 +39,8 @@ There are other rules and restrictions not shared here for the sake of spam and
 | flags?        | integer   | the [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account                         | identify              |
 | premium_type? | integer   | the [type of Nitro subscription](#DOCS_RESOURCES_USER/user-object-premium-types) on a user's account | identify              |
 | public_flags? | integer   | the public [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account                  | identify              |
+| bio?          | ?string   | the user's bio                                                                                       | identify              |
+| banner?       | ?string   | the user's banner hash                                                                               | identify              |
```

Comment:
Not very descriptive, how do you use it?

Reply:
You can't use it yet because no one knows for what it will be used for, however those two fields are already present in `v9`'s API responses.

Reply:
Then this PR is completely useless.